tox.ini

```
@@ -0,0 +1,12 @@
+[tox]
+envlist =
+    py{311, 310, 39}
+skipsdist = true
+
+[testenv]
+passenv =
+    FORCE_COLOR
+deps =
+    -rrequirements.txt
```

```suggestion
    -r requirements.txt
```

Easier to read at a glance with a space, and doesn't read like a typo, assuming it doesn't have to be elided

---

Unfortunately there's a bug in tox-ini-fmt that changes `-r requirements.txt` to `-r`:

https://github.com/tox-dev/tox-ini-fmt/issues/49

Shall we remove tox-ini-fmt or keep `-rrequirements.txt`?

---

Your call, but given it is a minor style nit and you were well aware of when deciding to add `tox-ini-fmt` (having reported said bug), I assume it would make sense to just keep it

---

Yeah, let's keep it in then.